jitsi/010-jvb-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: jvb
+  name: jvb-udp
+  namespace: jitsi
+spec:
+  type: NodePort
```

Comment:
This should be a ClusterIP since nodes should be considered ephemeral. You can use a UDP service with Ingress NGINX: https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/ https://github.com/hashbang/gitops/blob/master/ingress-nginx/tcp-services-cm.yaml and https://github.com/hashbang/gitops/blob/master/ingress-nginx/controller/resources.yaml#L17

You can include the changes to the ingress-nginx configuration in this PR to allow using Ingress NGINX for UDP inbound connections. DNS will also be managed automatically for this, since it's already done by the Ingress resource.